etstool.py

```diff
@@ -199,12 +203,12 @@ def install(edm, runtime, toolkit, environment, editable, source):
     elif toolkit == "wx":
         if sys.platform == "darwin":
             commands.append(
-                "{edm} run -e {environment} -- python -m pip install wxPython<4.1"
+                "{edm} run -e {environment} -- python -m pip install wxPython<4.1"  # noqa: E501
             )
         elif sys.platform == "linux":
             # XXX this is mainly for TravisCI workers; need a generic solution
             commands.append(
-                "{edm} run -e {environment} -- pip install -f https://extras.wxpython.org/wxPython4/extras/linux/gtk3/ubuntu-14.04/ wxPython<4.1"
+                "{edm} run -e {environment} -- pip install -f https://extras.wxpython.org/wxPython4/extras/linux/gtk3/ubuntu-14.04/ wxPython<4.1"  # noqa: E501
```

Comment:
this is tangential but we're not using the ubuntu 14.04 images on travis (i think xenial is 16.04) at the moment so i don't know if we should be using this url to download the expython wheels.

Reply:
yep xenial is 16.04 - https://docs.travis-ci.com/user/reference/linux#overview

Reply:
Yep, probably needs a different image.  Not in this PR though.